### PR TITLE
AD-1008/Modified OTA_HotelProductNotifRQ and common structs

### PR DESCRIPTION
**Pull request by eastern-aditya**
- Updated OTA_HotelProductNotifRQ and common structs based on Shiji Documentations, Shiji Message samples 
   & `cmm-support-scripts` [PR #31](https://github.com/Bidroombv/cmm-support-scripts/pull/31)

### CODE REVIEW

File: otamodel/common.go [hunk 1]
```diff
@@ -282,3 +285,45 @@ type AcceptedPayment struct {
 type PaymentCard struct {
 	CardCode string `json:"CardCode,attr"`
 }
+
+//POS **
+type POS struct {
```
Comment (eastern-rohini) [inline otamodel/common.go]: POS is not required

File: otamodel/OTA_HotelProductNotifRQ.go [hunk 2]
```diff
@@ -71,57 +131,65 @@ type StayDateRange struct {
...
+	End              string                `json:"End,omitempty"`
+	Mon              string                `json:"Mon,omitempty"`
+	Tue              string                `json:"Tue,omitempty"`
+	Weds             string                `json:"Weds,omitempty"`
+	Thur             string                `json:"Thur,omitempty"`
+	Fri              string                `json:"Fri,omitempty"`
+	Sat              string                `json:"Sat,omitempty"`
+	Sun              string                `json:"Sun,omitempty"`
+	PaymentCode      string                `json:"PaymentCode,omitempty"`
+	AcceptedPayments *AcceptedPayments     `json:"AcceptedPayments,omitempty"`
```
Comment (eastern-rohini) [inline otamodel/OTA_HotelProductNotifRQ.go]: OTA_HotelProductNotifRQ message received from CM not supporting AcceptedPayments struct. 
Please remove it.

File: otamodel/common.go (hunk 1 above)
Comment (eastern-aditya) [reply otamodel/common.go]: Done

File: otamodel/OTA_HotelProductNotifRQ.go (hunk 2 above)
Comment (eastern-aditya) [reply otamodel/OTA_HotelProductNotifRQ.go]: Done

**Review by eastern-rohini — APPROVED** [review]

File: otamodel/OTA_HotelProductNotifRQ.go [hunk 3]
```diff
@@ -71,57 +131,64 @@ type StayDateRange struct {
...
 }
 
 //TPAExtensions **
 type TPAExtensions struct {
-	PrepaymentPolicy PrepaymentPolicy `json:"PrepaymentPolicy,omitempty"`
+	PrepaymentPolicy *PrepaymentPolicy `json:"PrepaymentPolicy,omitempty"`
 }
 
 //GuaranteePayment **
 type GuaranteePayment struct {
```
Comment (eastern-yusuf) [inline otamodel/OTA_HotelProductNotifRQ.go]: I think same object is already declared in common.go with type PaymentPoliciesGuaranteePayment

**Review by eastern-yusuf — CHANGES_REQUESTED** [review]
small fix and we are good to go

File: otamodel/OTA_HotelProductNotifRQ.go (hunk 3 above)
Comment (eastern-aditya) [reply otamodel/OTA_HotelProductNotifRQ.go]: Modified

**Review by eastern-yusuf — APPROVED** [review]

**Pull request merged**